qa/common/gen_qa_model_repository

```diff
@@ -59,16 +59,25 @@ HOST_ZERODESTDIR=/tmp/qa_zero_model_repository
 HOST_RESHAPEDESTDIR=/tmp/qa_reshape_model_repository
 HOST_SEQDESTDIR=/tmp/qa_sequence_model_repository
 HOST_VARSEQDESTDIR=/tmp/qa_variable_sequence_model_repository
+HOST_ENSEMBLEDESTDIR=/tmp/qa_ensemble_model_repository
 
 rm -fr $HOST_DESTDIR $HOST_VARDESTDIR
 rm -fr $HOST_ZERODESTDIR $HOST_RESHAPEDESTDIR
 rm -fr $HOST_SEQDESTDIR $HOST_VARSEQDESTDIR
+rm -fr $HOST_ENSEMBLEDESTDIR
 mkdir -p $HOST_DESTDIR
 mkdir -p $HOST_VARDESTDIR
 mkdir -p $HOST_ZERODESTDIR
 mkdir -p $HOST_RESHAPEDESTDIR
 mkdir -p $HOST_SEQDESTDIR
 mkdir -p $HOST_VARSEQDESTDIR
+# Since the models required by ensemble models may not be available
+# at this point, storing ensemble models separately so that other qa directories
+# are proper model stores
+mkdir -p $HOST_ENSEMBLEDESTDIR/qa_model_repository
+mkdir -p $HOST_ENSEMBLEDESTDIR/qa_variable_model_repository
```

Comment:
You don't seem to generate/use variable, zero, or reshape ensemble repositories. Is that coming later?

Reply:
Yes. The models in both reshape and zero are essentially nop, so I am going to add them in a different PR since they are different from infer / variable in this sense.